File: 백준/Bronze/2525. 오븐 시계/오븐 시계.java
import java.io.BufferedReader;
import java.io.BufferedWriter;
import java.io.IOException;
import java.io.InputStreamReader;
import java.io.OutputStreamWriter;
import java.util.StringTokenizer;

public class Main {

	public static void main(String[] args) throws IOException {
		
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));
		
		StringTokenizer st = new StringTokenizer(br.readLine());
		
		int a = Integer.parseInt(st.nextToken());
		int b = Integer.parseInt(st.nextToken());
		int c = Integer.parseInt(br.readLine());
		
		int time = b + c;
		
		if(time >= 60) {
			a = a + (time / 60);
			time = time - 60 * (time / 60);
			
			if(a >= 24) {
				a = a - 24;
			}
		}
		
		System.out.println(a + " " + time);
		
		bw.close();
		
		
	}
		
}

File: 백준/Bronze/2525. 오븐 시계/오븐 시계.cc
#include <iostream>

using namespace std;

int main()
{
	int h, m, c;

	cin >> h >> m;
	cin >> c;

	int time = m + c;

	if (time >= 60) {
		h = h + (time / 60);
		time = time - 60 * (time / 60);

		if (h >= 24)
			h -= 24;
	}

	cout << h << " " << time;
	return 0;
}